nixos/modules/services/misc/plex.nix

```diff
@@ -30,6 +30,30 @@ in
         default = "plex";
         description = "Group under which Plex runs.";
       };
+
+
+      managePlugins = mkOption {
```

Comment:
Doing so would also automatically disable it when `extraPlugins` is removed from the config, which would still cause Nix to leave the last plugin symlinked in.